### PR TITLE
CORE-7692: Update Rails 5+ compatibility

### PR DESCRIPTION
**Pull request by wpdavenport**
Update openid_connect  dependency to Rails 5 compatibility

### CODE REVIEW

File: lib/omniauth/openid_connect/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module OpenIDConnect
-    VERSION = "0.2.2"
+    VERSION = "0.2.4"
```
Comment (tonycchung) [inline lib/omniauth/openid_connect/version.rb]: any reason we're skipping 3?

Comment (wpdavenport) [reply lib/omniauth/openid_connect/version.rb]: um no. I thought I had 3. It's a typo. thx

**Review by tonycchung — COMMENTED** [review]
perhaps we should add our own changelog as well.

would add something to the PR title as well

**Comment by tonycchung** [comment]
can we also add a `Gemfile.lock` to this gem

**Comment by wpdavenport** [comment]
This PR not necessary. :D 

Tested this with Rails 5.0  & Rails 5.1 apps and had no issues.

**Pull request closed without merging**